invenio_app_rdm/communities_ui/templates/semantic-ui/invenio_communities/details/browse/index.html

```diff
@@ -0,0 +1,34 @@
+{# -*- coding: utf-8 -*-
```

Comment:
nit: not something to change right now, but in the future the browse page could be made more modular. E.g. it could be responsible for organizing multiple sections, while the rendering of each section can be handled by the implementation.

Since our current use case is to display both collections and subcommunities in the same space, this approach works well for now. We can always adjust it in the future if needed.